herbert_core/classes/MPIFramework/messages/@aMessage/private/parce_payload_.m

```diff
@@ -12,40 +12,39 @@
 %   payload_p  -- modified input, containing all serializable classes
 
 
+% $$$     if iscell(payload)
+% $$$         payload_p  = cellfun(@(c)parce_payload_(c),payload,...
```

Comment:
And for this block, maybe just delete rather than comment out? otherwise say why you wish to retain a record of removed code